SAMLPortal/Client/src/js/materialize_js.js

```diff
@@ -0,0 +1,5 @@
+import 'materialize-css';
+
+document.addEventListener('DOMContentLoaded', function() {
+    M.AutoInit();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5434886)